#
#  ### Params of the orthopus addon ###
#
sampling_frequency          : 100     # Hz
direction                   : 1      # Invert motor controll direction
twin_button_up_pin          : 24
twin_button_down_pin        : 25
power_button_pin            : # TODO check which pin
limit_switch_pin            : # TODO check which pin
red_led_pin                 : # TODO check which pin
gree_led_pin                : # TODO check which pin
blue_led_pin                : # TODO check which pin
twin_button_velocity        : 8.0
actuator_address            : 19
close_loop_control          : false
